src/Legacy/Numbers/Words/Locale/Es.php

```diff
@@ -94,7 +94,8 @@ class Es extends Words
         'XAF' => [['franco CFA', 'francos CFA'], ['céntimo']],
         'XOF' => [['franco CFA', 'francos CFA'], ['céntimo']],
         'YUM' => [['dinar', 'dinares'], ['para']],
-        'ZAR' => [['rand'], ['cent']]
+        'ZAR' => [['rand'], ['cent']],
+        'COP' => [['peso', 'pesos'], ['centavo', 'centavos']]
```

Comment:
Could you please keep the alphabetical order?

Reply:
Alphabetical order applied